core/plannernotes.c

```diff
@@ -65,22 +116,16 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 		"PLAN DIVES SIMPLY BASED ON THE RESULTS GIVEN HERE."), deco);
 	disclaimer = buf;
 
-	if (!dp) {
-		free((void *)buffer);
-		free((void *)temp);
-		return;
-	}
+	if (!dp)
+		goto FINISHED;
```

Comment:
i don't think we have an exact coding style rules about labels, but i would make `FINISHED` lowercase.

Reply:
done.